docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>wrt lists : maybe mention that you should modify lists after assigning them, otherwise we don't handle them correctly (that might be fixed by now, though)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>I saw this in the issue on Thursday, I wasn't aware in fact but agreed we should say something about it. However I'm not sure I know what the right pattern is, what would you recommend we explain here?</p>

Reply:
Let's see how #524 gets resolved. Maybe point to that issue for the interested reader in the meantime.